engine/src/object.h

```diff
@@ -922,6 +922,14 @@ class MCObject : public MCDLlist
     //   a widget.
     virtual bool haswidgets(void);
     
+	// IM-2016-07-06: [[ Bug 17690 ]] Return the minimum stack file version that can fully
+	//   encode this object and its properties.
+	virtual uint32_t getminimumstackfileversion(void);
+	
+	// IM-2016-07-06: [[ Bug 17690 ]] Returns the minimum stack file version of this object
+	//   and any child objects.
+	uint32_t geteffectiveminimumstackfileversion(void);
```

Comment:
I think both these methods could be `const` since they only inspect the object rather than modifying it, e.g.

```C++
virtual uint32_t getminimumstackfileversion() const;
```